.vib/grafana-loki/cypress/cypress/support/index.js

```diff
@@ -0,0 +1,20 @@
+// ***********************************************************
+// This example support/index.js is processed and
+// loaded automatically before your test files.
+//
+// This is a great place to put global configuration and
+// behavior that modifies Cypress.
+//
+// You can change the location of this file or turn off
+// automatically serving support files with the
+// 'supportFile' configuration option.
+//
+// You can read more here:
+// https://on.cypress.io/configuration
+// ***********************************************************
+
+// Import commands.js using ES2015 syntax:
+import './commands';
```

Comment:
In fact, you could remove the file (support/commands.js) but you also need to remove this import statement.

Reply:
Oh OK! Excellent catch, On my way!
I forgot I've had this in the script.